Tests/BrowserServicesKitTests/PrivacyConfig/AppPrivacyConfigurationTests.swift

```diff
@@ -261,4 +273,66 @@ class AppPrivacyConfigurationTests: XCTestCase {
         XCTAssertFalse(config.isFeature(.gpc, enabledForDomain: "example.com"))
         XCTAssertFalse(config.isFeature(.gpc, enabledForDomain: "unp.com"))
     }
+    
+    let exampleVersionConfig =
+    """
+    {
+        "features": {
+            "gpc": {
+                "state": "enabled",
+                "exceptions": [
+                    {
+                        "domain": "example.com",
+                        "reason": "site breakage"
+                    }
+                ]
+            },
+            "trackingParameters": {
+                "state": "enabled",
+                "minSupportedVersion": "0.22.2",
+                "exceptions": []
+            },
+            "ampLinks": {
+                "state": "enabled",
+                "minSupportedVersion": "7.66.1.0",
+                "exceptions": []
+            }
+        },
+        "unprotectedTemporary": [
+            {
+                "domain": "unp.com",
+                "reason": "site breakage"
+            }
+        ]
+    }
+    """.data(using: .utf8)!
+    
+    func testMinSupportedVersionCheckReturnsCorrectly() {
+        var appVersion = MockAppVersionProvider(appVersion: "0.22.2")
+        
+        let mockEmbeddedData = MockEmbeddedDataProvider(data: exampleVersionConfig, etag: "test")
+        let mockProtectionStore = MockDomainsProtectionStore()
+
+        let manager = PrivacyConfigurationManager(fetchedETag: nil,
+                                                  fetchedData: nil,
+                                                  embeddedDataProvider: mockEmbeddedData,
+                                                  localProtection: mockProtectionStore)
+
+        let config = manager.privacyConfig
+        
+        XCTAssertTrue(config.isEnabled(featureKey: .gpc, versionProvider: appVersion))
+        XCTAssertTrue(config.isEnabled(featureKey: .trackingParameters, versionProvider: appVersion))
+        
+        appVersion = MockAppVersionProvider(appVersion: "0.22.3")
+        XCTAssertTrue(config.isEnabled(featureKey: .trackingParameters, versionProvider: appVersion))
```

Comment:
Would be good to get some tests in here that verify that app versions like `1.0.0` and `0.23.0` etc are valid too 🙂